### PR TITLE
Import ChaiLove

### PR DESCRIPTION
**Pull request by garbear**
See https://github.com/kodi-game/game.libretro/issues/32

### CODE REVIEW

**Comment by RobLoach** [comment]
Thanks a lot! :+1: 

The CMake files haven't been updated yet, so hold off on merging until that's done:
https://github.com/RobLoach/game.libretro.chailove/tree/master/depends

**Comment by garbear** [comment]
The CMake files can be generated with our kodi-game-scripting infrastructure if you fill out the config.py file: https://github.com/fetzerch/kodi-game-scripting/blob/master/kodi_game_scripting/config.py

**Comment by garbear** [comment]
What's left to do?

**Comment by RobLoach** [comment]
Updated the URL over at https://github.com/kodi-game/repo-binary-addons/pull/5 and https://github.com/kodi-game/game.libretro.chailove/pull/2

File: game.libretro.mrboom/platforms.txt [hunk 1]
```diff
@@ -1 +1 @@
-!windowsstore
```
Comment (RobLoach) [inline game.libretro.mrboom/platforms.txt:1]: Unrelated change?

File: game.libretro.chailove/platforms.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+all
```
Comment (RobLoach) [inline game.libretro.chailove/platforms.txt:1]: ChaiLove currently doesn't build on Android, or iOS. It works on Mac, Windows, Linux and ARM. Unsure if that affects this change.

Comment (garbear) [reply game.libretro.chailove/platforms.txt:1]: We'll leave it `all` for now, so the build failures will remind us to update it

**Comment by RobLoach** [comment]
Cleaned up the history in https://github.com/kodi-game/repo-binary-addons/pull/6 .

**Comment by garbear** [comment]
You could have rebased this PR and force-pushed instead of opening a new one.

**Comment by RobLoach** [comment]
You're right. You're more fancy at git than I... Probably also could've just cherry-picked the commits in, but alas. Thanks a lot!

**Comment by fetzerch** [comment]
Is there still work to be done for this? seems like the changes went in through different PRs.

**Comment by RobLoach** [comment]
Correct, I believe this can be closed safely. Thanks.
